pom.xml

```diff
@@ -143,6 +143,11 @@
                 <artifactId>org.wso2.carbon.um.ws.api</artifactId>
                 <version>${um.ws.api.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.wso2.tomcat</groupId>
+                <artifactId>tomcat-servlet-api</artifactId>
+                <version>${tomcat-servlet-api.version}</version>
+            </dependency>
```

Comment:
I think this dependency is not needed. This should be possible with java servlet api.